docs/G-Codes.md

```diff
@@ -1415,3 +1422,34 @@ command will probe the points specified in the config and then make independent
 adjustments to each Z stepper to compensate for tilt. See the PROBE command for
 details on the optional probe parameters. The optional `HORIZONTAL_MOVE_Z`
 value overrides the `horizontal_move_z` option specified in the config file.
+
+### [temperature_probe]
+
+The following commands are available when a
+[temperature_probe config section](Config_Reference.md#temperature_probe)
+is enabled.
+
+#### PROBE_DRIFT_CALIBRATE
```

Comment:
What about `TEMPERATURE_PROBE_CALIBRATION` instead of `PROBE_DRIFT_CALIBRATE`?  Putting the module name in the command helps reduce the chance of command conflicts in the future.

Reply:
I'm good with that.